### PR TITLE
69145 update fullscreen fotorama view

### PR DESCRIPTION
**Pull request by anqaka**
internal DEV-69145

### CODE REVIEW

File: styles/vendor/fotorama/modules/_fullscreen-icon.scss [hunk 1]
```diff
@@ -10,12 +10,23 @@
     height: $fotorama__block-size;
     background-image: $fotorama__close-icon;
     background-position: center;
+    background-color: $color-primary;
```
Comment (Januszpl) [inline styles/vendor/fotorama/modules/_fullscreen-icon.scss:13]: not better to use `$fotorama__bg-color` ?

File: web/images/fotorama/close.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill-rule="nonzero" d="M18.3 5.71a.996.996 0 0 0-1.41 0L12 10.59 7.11 5.7A.996.996 0 1 0 5.7 7.11L10.59 12 5.7 16.89a.996.996 0 1 0 1.41 1.41L12 13.41l4.89 4.89a.996.996 0 1 0 1.41-1.41L13.41 12l4.89-4.89c.38-.38.38-1.02 0-1.4z"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill-rule="nonzero" fill="#fff" d="M18.3 5.71a.996.996 0 0 0-1.41 0L12 10.59 7.11 5.7A.996.996 0 1 0 5.7 7.11L10.59 12 5.7 16.89a.996.996 0 1 0 1.41 1.41L12 13.41l4.89 4.89a.996.996 0 1 0 1.41-1.41L13.41 12l4.89-4.89c.38-.38.38-1.02 0-1.4z"/></svg>
```
Comment (Januszpl) [inline web/images/fotorama/close.svg:1]: Shouldn't we style it via css?

File: styles/vendor/fotorama/modules/_fullscreen-icon.scss (hunk 1 above)
Comment (anqaka) [reply styles/vendor/fotorama/modules/_fullscreen-icon.scss:13]: yup, good catch, thx

File: web/images/fotorama/close.svg (hunk 2 above)
Comment (anqaka) [reply web/images/fotorama/close.svg:1]: right, I used svg-uri mixin

**Pull request merged**